apps/demo/src/index.tsx

```diff
@@ -1,17 +1,14 @@
-import 'react-app-polyfill/stable';
-
 import { StrictMode } from 'react';
 import ReactDOM from 'react-dom';
 
-import 'normalize.css';
 import './styles.css';
 
 import DemoApp from './DemoApp';
 
-if (process.env.REACT_APP_DIST === 'true') {
-  // Import distributed lib/app styles (can't use `import()` easily due to asynchronicity and lack of top-level `await` support)
-  require('@h5web/app/dist/style-lib.css'); // eslint-disable-line @typescript-eslint/no-require-imports
-  require('@h5web/app/dist/style.css'); // eslint-disable-line @typescript-eslint/no-require-imports
+if (import.meta.env.VITE_DIST === 'true') {
+  // Import distributed lib/app styles
+  import('@h5web/app/dist/style-lib.css');
+  import('@h5web/app/dist/style.css');
```

Comment:
It seems that Vite works better than CRA with regard to importing stylesheets conditionally/dynamically. The asynchronicity issue mentioned in the comment is nowhere to be seen.

Reply:
This part prevents me from running the demo...

I get an error `Internal server error: Failed to resolve import "@h5web/app/dist/style-lib.css" from "src/index.tsx".` even if `import.meta.env.VITE_DIST` is `false`

I also get an inline eslint error in VSCode: `dynamic imports require a leading comment with the webpack chunknameeslintimport/dynamic-import-chunkname`

Reply:
Possibly linked to https://github.com/vitejs/vite/issues/6007

Reply:
These two imports are no longer required and have been removed.